presto-native-execution/presto_cpp/main/common/Counters.cpp

```diff
@@ -181,6 +181,35 @@ void registerPrestoCppCounters() {
   REPORT_ADD_STAT_EXPORT_TYPE(
       kCounterSsdCacheCumulativeReadCheckpointErrors,
       facebook::velox::StatType::AVG);
+  // Disk spilling stats.
+  REPORT_ADD_STAT_EXPORT_TYPE(
+      kCounterSpilledFiles,
+      facebook::velox::StatType::SUM);
```

Comment:
What is the reason we use SUM instead of AVG? Do we need 600 second sum aggr and 3600 second sum aggr? Thanks

Reply:
We want to measure the sum over some period but not average